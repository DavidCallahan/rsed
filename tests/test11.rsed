print "test 11a"
print 'test 11b'
print "test \"11c\""
print 'test \'11d\''
