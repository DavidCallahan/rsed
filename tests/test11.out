test 11a
test 11b
test "11c"
test '11d'
